test-suite/js/src/test/require-modules/org/scalajs/testsuite/jsinterop/ModulesTest.scala

```diff
@@ -197,4 +203,16 @@ object ModulesTest {
   @js.native
   @JSImport("querystring", JSImport.Namespace)
   object ExistentSubObject extends NonExistentSuperClass
+
+  /* #4268 Test that a super-class only used in an extends from a non-native JS
+   * class *is* imported.
+   */
+  @js.native
+  @JSImport("../test-classes/module-natives.js", "ModuleTestNativeParentClass")
```

Comment:
Since we're using our own, we should use a much more specific file name. This test would become moot if we added something else `module-natives.js` that we would import somewhere else. So the file name should be very specific to this test, so that we never reuse it for any other test.

Reply:
Good point!